src/app.ts

```diff
@@ -207,77 +212,90 @@ const addProfile = async (
     createProfile(profile);
     console.log(chalk.green(`Added new profile '${name}'`));
   } else {
-    const profileAnswers = await prompts([
-      {
-        type: 'text',
-        name: 'profile',
-        message: 'Name',
-      },
-      {
-        type: 'text',
-        name: 'accessKey',
-        message: 'Access key',
-      },
-      {
-        type: 'text',
-        name: 'secretKey',
-        message: 'Secret key',
-      },
-      {
-        type: 'number',
-        name: 'keyMaxAge',
-        message: 'Secret key maximum age in days',
-        initial: 90,
-      },
-      {
-        type: 'text',
-        name: 'defaultRegion',
-        message: 'Default region',
-      },
-      {
-        type: 'select',
-        name: 'outputFormat',
-        message: 'Output format',
-        choices: [
-          { title: 'json', value: 'json' },
-          { title: 'yaml', value: 'yaml' },
-          { title: 'text', value: 'text' },
-          { title: 'table', value: 'table' },
-        ],
-        initial: 1,
-      },
-      {
-        type: 'confirm',
-        name: 'useMfa',
-        message: 'Use MFA',
-      },
-    ]);
+    const profileAnswers = await prompts(
+      [
+        {
+          type: 'text',
+          name: 'profile',
+          message: 'Name',
+        },
+        {
+          type: 'text',
+          name: 'accessKey',
+          message: 'Access key',
+        },
+        {
+          type: 'text',
+          name: 'secretKey',
+          message: 'Secret key',
+        },
+        {
+          type: 'number',
+          name: 'keyMaxAge',
+          message: 'Secret key maximum age in days',
+          initial: 90,
+        },
+        {
+          type: 'text',
+          name: 'defaultRegion',
+          message: 'Default region',
+        },
+        {
+          type: 'select',
+          name: 'outputFormat',
+          message: 'Output format',
+          choices: [
+            { title: 'json', value: 'json' },
+            { title: 'yaml', value: 'yaml' },
+            { title: 'text', value: 'text' },
+            { title: 'table', value: 'table' },
+          ],
+          initial: 1,
+        },
+        {
+          type: 'confirm',
+          name: 'useMfa',
+          message: 'Use MFA',
+        },
+      ],
+      { onCancel: () => onCancel('Aborting [add-profile]') }
```

Comment:
```suggestion
      { onCancel: () => onCancel('Cancelling add profile') }
```

Update the rest of these messages to match this format